### PR TITLE
Update maintainers

### PR DESCRIPTION
**Pull request by ivanpauno**
The ROS 2 team is distributing maintenance responsibilities.
Before we used to triage issues across all repositories in a join effort of the ROS 2 team, the idea now is that one person (or a small group) keep tracks of issues in a particular repo and triage them.

I'm adding me as a maintainer here, as I'm assigned to this repo.

cc @eboasson This shouldn't make a difference for ADLINK people, it's only a reorganization in the ROS 2 team triaging process.

### CODE REVIEW

**Review by clalancette — APPROVED** [review]
Seems reasonable to me, but let's wait for an approval from @eboasson before merging.

**Review by eboasson — APPROVED** [review]
It is my pleasure to click “approve” :)

**Pull request merged**
